common/remote/sockfile.cpp

```diff
@@ -986,7 +990,7 @@ class CRemoteBase: public CInterface
                 }
                 else
                     socket.setown(ISocket::connect(ep));
-                if (useSSL)
+                if (useSSL == SSLOnly)
```

Comment:
It does not - that is the whole key to this.  If Dafilesrv accepts both and you the client are allowed to connect non-ssl then that is what you do.  The client uses the 'best' port it is allowed to use, so if its both that means non-ssl.